storage-rocketmq/src/main/java/org/apache/rocketmq/schema/registry/storage/rocketmq/RocketmqClient.java

```diff
@@ -203,39 +200,28 @@ public void startRemoteStorage() {
         try {
             producer.start();
 
-            scheduleConsumer.setPullThreadNums(4);
+            scheduleConsumer.subscribe(storageTopic, "*");
+            scheduleConsumer.registerMessageListener(new MessageListener());
             scheduleConsumer.start();
-
-            Collection<MessageQueue> messageQueueList = scheduleConsumer.fetchMessageQueues(storageTopic);
-            scheduleConsumer.assign(messageQueueList);
-            messageQueueList.forEach(mq -> {
-                try {
-                    scheduleConsumer.seekToBegin(mq);
-                } catch (MQClientException e) {
-                    e.printStackTrace();
-                }
-            });
-            this.scheduledExecutorService.scheduleAtFixedRate(new RocketmqStoragePullTask(),
-                0, PULL_TASK_INTERVAL, TimeUnit.MILLISECONDS);
-
         } catch (MQClientException e) {
             throw new SchemaException("Rocketmq client start failed", e);
         }
     }
 
-    public class RocketmqStoragePullTask implements Runnable {
+    public class MessageListener implements MessageListenerConcurrently {
 
```

Comment:
It's better to use MessageListenerOrderly instead of MessageListenerConcurrently